File: Assignments/Week_01/swapArray.js
const swapArrays = (arrOne, arrTwo) => {
	let temp1 = arrOne.slice()
	let temp2 = arrTwo.slice()
	arrOne = []
	arrTwo = []
	arrOne = temp2.slice()
	arrTwo = temp1.slice()
	console.log(arrOne,arrTwo)
}

let a = [1,2,3,4,56,"a"]
let b = [4,3,44,11,2]

swapArrays(a,b)

File: Assignments/Week_01/swapArray.py
def swapArrays(arrOne,arrTwo):
	temp1 = arrOne
	arrOne = arrTwo
	arrTwo = temp1
	return(arrOne,arrTwo)


a = [1,2,3,4,56,"a"]
b = [4,3,44,11,2]
print(swapArrays(a,b))
